docs/tutorial/work-pools.md

```diff
@@ -0,0 +1,306 @@
+
+- You can configure default infrastructure configurations on your work pools that all jobs inherit and can override.
+- Platform teams can use work pools to expose opinionated (and enforced!) interfaces to the infrastructure that they oversee.
+- Work pools can be used to prioritize (or limit) flow runs through the use of [work queues](/concepts/work-pools/#work-queues).
+
+Prefect provides several [types of work pools](/concepts/work-pools/#work-pool-types).
+Prefect Cloud provides a Prefect Managed work pool option that is the simplest way to deploy remote workflows.
+A cloud-provider account, such as AWS, is not required with a Prefect Managed work pool.
+
+## Set up a work pool
+
+### Create a Prefect Managed work pool
+
+In your terminal, run the following command to set up a work pool named `my-managed-pool` of type `prefect:managed`.
+
+<div class="terminal">
+
+```bash
+prefect work-pool create my-managed-pool --type prefect:managed 
+```
+
+</div>
+
+Let’s confirm that the work pool was successfully created by running the following command.
+
+<div class="terminal">
+
+```bash
+prefect work-pool ls
+```
+
+</div>
+
+You should see your new `my-managed-pool` in the output list.
+
+Finally, let’s double check that you can see this work pool in the UI.
+
+Navigate to the **Work Pools** tab and verify that you see `my-managed-pool` listed.
+
+Feel free to select **Edit** from the three-dot menu on right of the work pool card to view the details of your work pool.
+
+Work pools contain configuration that is used to provision infrastructure for flow runs.
+For example, you can specify additional Python packages or environment variables that should be set for all deployments that use this work pool.
+Note that individual deployments can override the work pool configuration.
+
+Now that you’ve set up your work pool, we can create a deployment that is tied to this work pool.
+Let's deploy your tutorial flow to `my-managed-pool`.
+
+## Create the deployment
+
+From our previous steps, we now have:
+
+1. [A flow](/tutorial/flows/)
+2. A work pool
+
+Let's update our `repo_info.py` file to create a deployment in Prefect Cloud.
+
+The updates that we need to make to `repo_info.py` are:
+
+1. Change `flow.serve` to `flow.deploy`.
+2. Tell `flow.deploy` which work pool to deploy to.
+
+Here's what the updated `repo_info.py` looks like:
+
+```python hl_lines="17-23" title="repo_info.py"
+import httpx
+from prefect import flow
+
+
+@flow(log_prints=True)
+def get_repo_info(repo_name: str = "PrefectHQ/prefect"):
+    url = f"https://api.github.com/repos/{repo_name}"
+    response = httpx.get(url)
+    response.raise_for_status()
+    repo = response.json()
+    print(f"{repo_name} repository statistics 🤓:")
+    print(f"Stars 🌠 : {repo['stargazers_count']}")
+    print(f"Forks 🍴 : {repo['forks_count']}")
+
+TK repo below
+
+if __name__ == "__main__":
+    get_repo_info.from_source(
+        source="https://github.com/desertaxle/demo.git", 
```

Comment:
Do we host this flow in a public repo under the PrefectHQ org? If not, I think we should to make this code runnable as is.

Reply:
I don't believe so and I think that's a good idea.

Reply:
Made it runnable and put in own repo.